forc-pkg/Cargo.toml

```
@@ -15,6 +15,7 @@ git2 = "0.14"
 petgraph = { version = "0.6", features = ["serde-1"] }
 semver = { version = "1.0", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
+serde_ignored = "0.1.2"
```

Can you use major.minor instead of major.minor.patch?

```suggestion
serde_ignored = "0.1"
```